### PR TITLE
Create intellij tricks doc

### PR DESCRIPTION
**Pull request by fiunchinho**

### CODE REVIEW

**Review by xh3b4sd — APPROVED** [review]
I would like to have this linked in the readme. Then I am fine.

**Comment by fiunchinho** [comment]
Fixed

File: go/intellij.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Configuring IntelliJ Idea
```
Comment (kopiczko) [inline go/intellij.md]:
```suggestion
# IntelliJ/Goland configuration
```

File: go/intellij.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+### Imports
```
Comment (kopiczko) [inline go/intellij.md]:
```suggestion
## Imports
```

File: go/intellij.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+Use goimports and format the imports on your code automatically using the "optimize imports" action.
+You can configure how the imports are arranged.
+![](images/imports.png)
+
+### Formatting
```
Comment (kopiczko) [inline go/intellij.md]:
```suggestion
## Formatting
```

**Comment by fiunchinho** [comment]
Added new lines after headings too.

**Review by kopiczko — APPROVED** [review]

**Review by whites11 — APPROVED** [review]
Lgtm

**Pull request merged**